sfxclient/wrapper.go

```diff
@@ -17,6 +17,7 @@ func NewMultiCollector(collectors ...Collector) Collector {
 }
 
 // Datapoints returns the datapoints from every collector.
+// nolint:prealloc
```

Comment:
why not pre alloc size of m? likely they'll be at least 1 dp per?

Reply:
done